application/src/main/java/org/togetherjava/tjbot/commands/moderation/ActionRecord.java

```diff
@@ -9,7 +9,7 @@
 /**
  * Record for actions as maintained by {@link ModerationActionsStore}. Each action has a unique
  * caseId.
- *
+ * 
```

Comment:
Why is there a space?

Reply:
spotless

Reply:
I doubt. The file was already fine with spotless before you touched it. Remove the space.